### PR TITLE
Publish Fineract Provider binary

### PR DESCRIPTION
**Pull request by IOhacker**
Publish Fineract Provider binary

## Description

Publish Fineract Provider binary required for having gradle/maven repositories needed by the development of Fineract plugins 

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Write the commit message as per https://github.com/apache/fineract/#pull-requests

- [ ] Acknowledge that we will not review PRs that are not passing the build _("green")_ - it is your responsibility to get a proposed PR to pass the build, not primarily the project's maintainers.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [ ] Follow coding conventions at https://cwiki.apache.org/confluence/display/FINERACT/Coding+Conventions.

- [ ] Add required Swagger annotation and update API documentation at fineract-provider/src/main/resources/static/legacy-docs/apiLive.htm with details of any API changes

- [ ] Submission is not a "code dump".  (Large changes can be made "in repository" via a branch.  Ask on the developer mailing list for guidance, if required.)

FYI our guidelines for code reviews are at https://cwiki.apache.org/confluence/display/FINERACT/Code+Review+Guide.

### CODE REVIEW

**Review by vidakovic — APPROVED** [review]
LGTM

**Comment by vidakovic** [comment]
@IOhacker Restarted failed builds... FYI.

**Comment by galovics** [comment]
@IOhacker this breaks the publish task for me locally and for openMF/fineract too. @vidakovic can you check it yourself? Thanks.

**Comment by francisguchie** [comment]
@vidakovic could this be the reason why the fineract-pentaho plugin is failing to publish too? see this [issue](https://github.com/openMF/fineract-pentaho/issues/85)

**Comment by vidakovic** [comment]
@galovics how can I test this? Do I need credentials to publish to the Maven repo?

**Comment by galovics** [comment]
> @galovics how can I test this? Do I need credentials to publish to the Maven repo?

@vidakovic don't bother with creds, just do a ./gradlew publish locally and it'll fail.

The error is:
```
> Failed to publish publication 'mavenJava' to repository 'apache'
   > Invalid publication 'mavenJava': POM file is invalid. Check any modifications you have made to the POM file.
```

cc @IOhacker

File: build.gradle [hunk 1]
```diff
@@ -38,7 +38,8 @@ buildscript {
         fineractPublishProjects = subprojects.findAll{
             [
                 'fineract-avro-schemas',
-                'fineract-client'
+                'fineract-client',
+                'fineract-provider'
```
Comment (galovics) [inline build.gradle:42]: The addition causes a failure for the publication for me and for the openMF/fineract repo.

**Review by galovics — CHANGES_REQUESTED** [review]

**Comment by IOhacker** [comment]
Closing this PR

**Pull request closed without merging**